db/migrate/20181105223440_add_started_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddStartedToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :started_at, :date
```

Comment:
Why are you including this in the same PR as the user's controller? They should be separate.

Reply:
You also might consider having an `up` and `down` method so you can rollback the db to easily remove the column if need be.

Reply:
gah - forgot to revert my local before opening a new pr